features/step_definitions/snippet_steps.rb

```diff
@@ -0,0 +1,23 @@
+Given(/^I am authenticated$/) do
+  user = User.new email: "landry.soules@gmail.com", password: "1Password", username: "landry"
+  signup user
+  authenticate user
+  page.assert_text "Signed in successfully."
```

Comment:
Thanks, sometimes I have hard times figuring out where cucumber finishes and where rspec starts !